testdata/expected/docker/java-error-tolerance/org.srclib.srclib-java/unresolved-maven-deps/JavaArtifact.depresolve.json

```diff
@@ -16,9 +19,17 @@
     "Raw": {
       "groupID": "some-group-that-doesnt-exist",
       "artifactID": "some-artifact-that-doesnt-exist",
-      "version": "1.2.3"
+      "version": "1.2.3",
+      "scope": "compile",
+      "type": "jar",
+      "repoURI": ""
     },
-    "Error": "Could not download file http://central.maven.org/maven2/some-group-that-doesnt-exist/some-artifact-that-doesnt-exist/1.2.3/some-artifact-that-doesnt-exist-1.2.3.pom"
```

Comment:
It seems like this test case was trying to detect that an error was correctly caught here. Was this regressed or what is the reason that this error should no longer appear?